apps/web/src/components/milestone-logs/recruitment/Milestone.tsx

```diff
@@ -31,7 +32,7 @@ const getInitialValues = (
   start_date: `${status?.start_date || ''}`,
   notes: `${status?.notes || ''}`,
   reason: `${status?.reason?.id || ''}`,
-  effective_date: `${status?.effective_date || new Date().toISOString()}`,
+  effective_date: `${status?.effective_date || ''}`,
```

Comment:
@coryatwater-FW 
I reverted this to be empty string because it is confusing when the form shouldn't present pre-populated date.
Instead, I added ValidateIf to DTOs.